.circleci/config.yml

```diff
@@ -171,14 +171,14 @@ commands:
 jobs:
   test:
     docker:
-      - image: circleci/node:9.8.0-stretch
+      - image: circleci/node:14-stretch
```

Comment:
changes in this file are only needed to make circleci build the app successfully